.travis.yml

```diff
@@ -85,8 +85,14 @@ before_install:
     - tools/check_sdist.py
 
 install:
-    - python setup.py develop
     - ccache --show-stats
+    # Test installing without cython using the sdist
+    - if [[ $INSTALL_FROM_SDIST ]]; then
+        pip uninstall cython -y;
+        pip install dist/scikit-image-*.tar.gz;
```

Comment:
Since you removed the setup.py develop above, where does the sdist get generated?

Reply:
`check_sdist` builds one. https://github.com/scikit-image/scikit-image/blob/master/tools/check_sdist.py#L11

Reply:
Ah. Perhaps it's better to break things up: `python setup.py sdist; tools/check_sdist.py`. It would make things much more readable...